scripts/runtime/VibeRuntime.Common.ps1

```
@@ -261,6 +261,130 @@ function Get-VibeHostContinuationContext {
+            (Test-VibeObjectHasProperty -InputObject $RuntimeInputPacket -PropertyName 'continuation_context') -and
+            $null -ne $RuntimeInputPacket.continuation_context -and
+            (Test-VibeObjectHasProperty -InputObject $RuntimeInputPacket.continuation_context -PropertyName 'revision_delta')
+        ) {
+            $items += @(Get-VibeNormalizedStringList -Values $RuntimeInputPacket.continuation_context.revision_delta)
+        }
+    }
+
+    return @($items | Where-Object { -not [string]::IsNullOrWhiteSpace([string]$_) } | Select-Object -Unique)
+}
+
+function Test-VibeRevisionDeltaRequestsAdvisoryOnlySpecialists {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Test-VibeRevisionDeltaRequestsAdvisoryOnlySpecialists' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Test-VibeRevisionDeltaRequestsAdvisoryOnlySpecialists'

(PSUseSingularNouns)